client/header/activity-panel/index.js

```diff
@@ -316,14 +327,6 @@ export const ActivityPanel = ( { isEmbedded, query, userPreferencesData } ) => {
 		}
 	};
 
-	const redirectToHomeScreen = () => {
```

Comment:
Just checking, as I see this this code block was moved. Did anything change in this function we need to be aware of?

Reply:
Actually, it's moved a little bit upwards since lint is nagging me with the function being called before it's defined 😬